modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/js/CommerceChannelSite.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
```

Comment:
Self-review: two copyrights??

I made this file by copying from somewhere else, not paying attention to jsdoc. Searching for it now, we have 10 files in DXP with double copyrights :facepalm:

Reply:
@wincent I opened a source-formatter bug for this: https://issues.liferay.com/browse/LPS-131310

Reply:
Great. Yeah, that'll happen whenever somebody copies from a `dxp/` directory to a non-`dxp/` directory, or vice versa. The ESLint plugin and rule (`notice/notice`) isn't super smart (we didn't write it — if we had, it would surely be 🧠), so all it does is a simple textual comparison: if it doesn't see the header it wants, it _adds_ it — it does not try to _replace_ an existing, wrong header.

If this happens a lot, we can try [the `onNonMatchingHeader` setting](https://www.npmjs.com/package/eslint-plugin-notice#onnonmatchingheader) (which I didn't use at first because it didn't seem super robust; that may have change by now), or submitting a feature request upstream, or we can just roll our own... I mean, we already have [24 in-house custom ESLint rules](https://github.com/liferay/liferay-frontend-projects/tree/master/projects/eslint-config), so we have plenty of examples (even just limiting it to our own rules) of how to do this.